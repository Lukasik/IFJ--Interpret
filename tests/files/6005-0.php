<?php
$x = 1;
$y = 6;
while ($x - $y)
{
	$x = $x + 1;
	if($x > 0)
	{
		$e = put_string($x);
		if($x > 2)
		{
			$e = put_string($x);
			if($x > 4)
			{
				$e = put_string($x);
			}
			else
			{
				$e = put_string($x);
			}
		}
		else
		{
			$e = put_string($x);
		}
	}
	else
	{
		$e = put_string($x);
	}
}
