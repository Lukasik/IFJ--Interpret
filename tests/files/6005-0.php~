<?php
$x = 1;
$y = 6;
while ($x - $y)
{
	$x = $x + 1;
	if($x > 0)
	{
		put_string($x);
		if($x > 2)
		{
			put_string($x);
			if($x > 4)
			{
				put_string($x);
			}
			else
			{
				put_string($x);
			}
		}
		else
		{
			put_string($x);
		}
	}
	else
	{
		put_string($x);
	}
}
