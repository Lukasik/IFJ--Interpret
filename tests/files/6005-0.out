22333444555666